sensors/baro/lps25xx.c

```diff
@@ -121,15 +125,22 @@ static int lps25xx_hwSetup(lps25xx_ctx_t *ctx)
 		return -1;
 	}
 
-	if (spiWriteReg(ctx, REG_RES_CONF, VAL_AVGT_8 | VAL_AVGP_128) < 0) {
+	if (spiWriteReg(ctx, REG_CTRL_REG2, VAL_BOOT) < 0) {
 		return -1;
 	}
+	usleep(1000 * 10);
 
 	if (spiWriteReg(ctx, REG_CTRL_REG1, (VAL_PD_ACTIVE_MODE | VAL_ODR_25HZ)) < 0) {
 		return -1;
 	}
 	usleep(1000 * 100);
 
+	if (spiWriteReg(ctx, REG_RES_CONF, VAL_AVGT_8 | VAL_AVGP_32) < 0) {
+		return -1;
+	}
+	usleep(1000 * 100);
+
+
```

Comment:
Can You add some comment above each `if` statement to describe what an option is written to sensor?